admin/js/comClasses.js

```diff
@@ -217,7 +217,8 @@ var comclasses = {
     0x2F: {name: 'ZIP_ADV_SERVICES',                       role: ''},
     0x2e: {name: 'ZIP_CLIENT',                             role: ''},
     0x24: {name: 'ZIP_SERVER',                             role: ''},
-    0x23: {name: 'ZIP_SERVICES',                           role: ''}
+    0x23: {name: 'ZIP_SERVICES',                           role: ''},
+    0x5B: {name: 'CENTRAL_SCENE',                          role: ''}

```

Comment:
We might want to double check if there is a definition for all command classes, see 
https://github.com/AlCalzone/node-zwave-js/blob/master/src/lib/commandclass/CommandClasses.ts#L4 for reference

Reply:
Thanks, remaining com classes added.